README.md

```diff
@@ -82,6 +82,32 @@ node bin/sourcecred.js load ipfs/js-ipfs ipfs/go-ipfs --output ipfs/meta-ipfs
 
 will create a graph called `ipfs/meta-ipfs` in the cred explorer, containing the combined contents of the js-ipfs and go-ipfs repositories.
 
+#### Docker
+
+The prototype may also be run using `docker-compose`. To build the container,
+
+```Bash
+git clone https://github.com/sourcecred/sourcecred.git
+cd sourcecred
+docker-compose build
+```
+
+Arguments can now be passed to `node bin/sourcecred.js load` via:
+
+```Bash
+docker-compose run --rm \
```

Comment:
Considering you already reference the env var and port in the compose file, why not recommend using a `.env` and `docker-compose up sourcecred`?

Reply:
yes and of course exporting the env var in the console should work as well, however I'm basing these instructions off the current non-docker documentation which keeps exporting the variable each time the command is run to be explicit.

If the old documentation is updated to talk about `.bashrc` and similar the it would make sense to talk about equivalent convenience functionality for `docker`...